### ISSUE TITLE
Daily Builds link is not up to date in README

### ISSUE THREAD

**Issue by eerhardt** [dotnet/runtime#32055]
https://github.com/dotnet/runtime/tree/95cff585613585e06a316e850830ef6e401858af#daily-builds

The whole "Master" column is out of date. The "latest" daily build is the last one from core-setup. I'm going to delete that whole table until we can get a way to keep this table up to date. (Note that the other columns don't apply to this repo, so there is no sense putting them here. The servicing branches are in core-setup.)

Leaving this issue open to track some new way of getting a latest "Daily Build" link on our REAME.

/cc @dagood

**Comment by dagood** [comment]
Currently the table is out of date since we didn't port the "latest" channel infra to the consolidated repo. I don't know a way to get info about the latest builds with public information, other than watching downstream repos for updates. (With Msft credentials, you can use Darc/BAR.)

Options:

* Wait for https://github.com/dotnet/arcade/issues/3963. 
* Port the "latest" channel infra. It makes the official build highly susceptible to breaks from Arcade, because adding/removing/changing channels becomes a breaking change.

@dotnet/runtime-infrastructure @mmitche 

---

> (Note that the other columns don't apply to this repo, so there is no sense putting them here. The servicing branches are in core-setup.)

dotnet/runtime is the place to file issues and track whatever's going on with core-setup servicing, so I think it makes *some* sense to have them here. We also got rid of `master` in core-setup, and the `archive` default branch simply points you here. There are no signposts to try looking at the servicing branches--if there were, maybe it would make sense to maintain the tables there. We produce the same set of artifacts here as we did in core-setup so it's not IMO significant that it's a different repo now.

(The POR has also been to use `master` and delete the tables from the release branches, but that's not an argument in itself. https://github.com/dotnet/runtime/issues/3302)

/cc @NikolaMilosavljevic

**Comment by Symbai** [comment]
Is there a way for developers to at least manually get the daily builds somewhere else?

**Comment by stephentoub** [comment]
https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries

**Comment by Symbai** [comment]
That's SDK bundle but I was referring to runtime only installers

**Comment by dagood** [comment]
A way to get it at the moment is look in this file for `MicrosoftNETCoreAppRefPackageVersion`:
https://github.com/dotnet/core-sdk/blob/master/eng/Versions.props#L72

E.g. `5.0.0-preview.1.20117.2`

Assemble the URL of the artifact you're interested in using that version number:

`https://dotnetcli.blob.core.windows.net/dotnet/Runtime/{version}/{name including version}`

Example:
https://dotnetcli.blob.core.windows.net/dotnet/Runtime/5.0.0-preview.1.20117.2/dotnet-runtime-5.0.0-preview.1.20117.2-win-x64.exe

To get a build that hasn't merged into Core-SDK yet, you can look to see if there's an active auto-upgrade PR and grab the version from that: https://github.com/dotnet/core-sdk/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Aapp%2Fdotnet-maestro+base%3Amaster

(Unfortunately not any way to look up build versions directly without Microsoft auth. https://github.com/dotnet/arcade/issues/1818.)

**Comment by Symbai** [comment]
@dagood It is a bit complicated but unfortunately I need runtime only of the preview, so thank you for helping me out ❤

**Comment by dagood** [comment]
This should be unblocked soon per https://github.com/dotnet/arcade/pull/4948.

https://github.com/dotnet/runtime/pull/33230 contains this arcade update: https://github.com/dotnet/arcade/compare/421b20fb860907c45673de791177e3fd61402197...d9a664e151f9973f1a3679f48d24c3407d77c9b2, which contains the commit from that PR. So, we're currently blocked on that auto-update.

Once this link suggested in the arcade PR works, I think we can move on this issue:
https://aka.ms/dotnet/net5/dev/Runtime/dotnet-apphost-pack-osx-x64.pkg

**Comment by dagood** [comment]
From an official build, looks like this is almost there and we got *some* links that work, but we hit this Arcade bug causing publish failure: https://github.com/dotnet/arcade/issues/5044

```
  The following aka.ms links for blobs will be created:
    aka.ms/dotnet/net5/dev/Runtime/dotnet-apphost-pack-osx-x64.pkg -> https://dotnetcli.blob.core.windows.net/dotnet/Runtime/5.0.0-preview.3.20161.13/dotnet-apphost-pack-5.0.0-preview.3.20161.13-osx-x64.pkg
    aka.ms/dotnet/net5/dev/Runtime/dotnet-host-internal-osx-x64.pkg -> https://dotnetcli.blob.core.windows.net/dotnet/Runtime/5.0.0-preview.3.20161.13/dotnet-host-internal-5.0.0-preview.3.20161.13-osx-x64.pkg
    aka.ms/dotnet/net5/dev/Runtime/dotnet-hostfxr-internal-osx-x64.pkg -> https://dotnetcli.blob.core.windows.net/dotnet/Runtime/5.0.0-preview.3.20161.13/dotnet-hostfxr-internal-5.0.0-preview.3.20161.13-osx-x64.pkg
    aka.ms/dotnet/net5/dev/Runtime/dotnet-runtime-osx-x64.pkg -> https://dotnetcli.blob.core.windows.net/dotnet/Runtime/5.0.0-preview.3.20161.13/dotnet-runtime-5.0.0-preview.3.20161.13-osx-x64.pkg
    aka.ms/dotnet/net5/dev/Runtime/dotnet-runtime-internal-osx-x64.pkg -> https://dotnetcli.blob.core.windows.net/dotnet/Runtime/5.0.0-preview.3.20161.13/dotnet-runtime-internal-5.0.0-preview.3.20161.13-osx-x64.pkg
    aka.ms/dotnet/net5/dev/Runtime/dotnet-targeting-pack-osx-x64.pkg -> https://dotnetcli.blob.core.windows.net/dotnet/Runtime/5.0.0-preview.3.20161.13/dotnet-targeting-pack-5.0.0-preview.3.20161.13-osx-x64.pkg
D:\a\1\s\.packages\microsoft.dotnet.arcade.sdk\5.0.0-beta.20160.4\tools\SdkTasks\PublishArtifactsInManifest.proj(79,5): error : The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.
...
```

**Comment by dagood** [comment]
@hoyosjs submitted a draft PR working on this: https://github.com/dotnet/runtime/pull/43392. Thanks! 👍
